tools/sim/README

```diff
@@ -0,0 +1,26 @@
+Needs Ubuntu 16.04
+
+== Checkout openpilot ==
+cd ~/
+git clone https://github.com/commaai/openpilot.git
+# Add export PYTHONPATH=$HOME/openpilot to your bashrc
+
+== Tensorflow (optional, for fast models) ==
+
+cd ~/openpilot/external/tensorflow
+./download.sh
+
+== Install (in tab 1) ==
+
+./start_carla.sh  # install CARLA 0.9.7 and start the server
+
+== openpilot (in tab 2) ==
+
+cd ~/openpilot/selfdrive/
+NOBOARD=1 ./manager.py
```

Comment:
For me it complained about Passive not being set. Should we add an env variable, e.g. `PASSIVE=0 ./manager.py` that writes the param?